### DISCUSSION TITLE
How to option back in to focus in refetchOnWindowFocus?

### DISCUSSION THREAD

**Discussion by personalcodecare** [TanStack/query#8279] [Q&A]
I'm having issue to keep some queries always update if the user goes to a second screen and gets back to the app window. 
Is there an easy way to use the same implementation as it was before? Instead of relying on workarounds?

**Comment by TkDodo** [comment]
you can implement your own event listener and listen to the `focus` event:

https://tanstack.com/query/latest/docs/reference/focusManager#focusmanagerseteventlistener
